### DISCUSSION TITLE
Ability to test value in column?

### DISCUSSION THREAD

**Discussion by titantwentyone** [filamentphp/filament#4020] [Help]
I might be missing something but is there an assertion to check that the value of a column is correct. E.g given:

```
Tables\Columns\TextColumn::make('no_of_products')
    ->getStateUsing(fn($record) => $record->products()->count()),
```
is there something available along the lines of...
```
livewire(ListCategories::class)
    ->assertTableColumnDataSet($category->id, 'no_of_products', 10);
```
Happy to PR if nothing available.

**Comment by danharrin** [comment]
Yeah I think that would be cool :)

**Reply by danharrin** [reply]
Not 100% sure what approach you would need to take though

**Reply by titantwentyone** [reply]
I actually just went ahead and did it - hopefully okay - will push PR :)

**Reply by titantwentyone** [reply]
#4022
